samples/client/petstore/scala-akka/src/main/scala/org/openapitools/client/api/PetApi.scala

```diff
@@ -20,7 +20,7 @@ import org.openapitools.client.core.ApiKeyLocations._
 
 object PetApi {
 
-  def apply(baseUrl: String = "http://petstore.swagger.io/v2") = new PetApi(baseUrl)
+  def apply(baseUrl: String = "https://petstore.swagger.io/v2") = new PetApi(baseUrl)
```

Comment:
please revert this change as the local instance of the petstore server doesn't support https